build.gradle.kts

```diff
@@ -1,23 +1,32 @@
-allprojects {
-    repositories {
-        jcenter()
-    }
-}
-
 plugins {
-    id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
+  id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
+  kotlin("jvm") version "1.5.10" apply false
 }
 
 subprojects {
-    version = "1.0"
+  version = "1.0"
 }
 
 nexusPublishing {
-    repositories {
-            sonatype {
-                packageGroup.set("io.github.alexanderprendota")
-                nexusUrl.set(uri("https://s01.oss.sonatype.org/service/local/"))
-                snapshotRepositoryUrl.set(uri("https://s01.oss.sonatype.org/content/repositories/snapshots/"))
-            }
+  repositories {
+    sonatype {
+      packageGroup.set("io.github.alexanderprendota")
+      nexusUrl.set(uri("https://s01.oss.sonatype.org/service/local/"))
+      snapshotRepositoryUrl.set(uri("https://s01.oss.sonatype.org/content/repositories/snapshots/"))
     }
+  }
+}
+
+subprojects {
+  apply {
+    plugin("org.jetbrains.kotlin.jvm")
+  }
+
+  dependencies {
+    val implementation by configurations
```

Comment:
val implementation by configurations why do u need it?

Reply:
Because of  type-safe model accessor.
https://docs.gradle.org/current/userguide/kotlin_dsl.html#sec:kotlin_using_standard_api
There is another way: ```"implementation"```